### PR TITLE
Created a prototype of every enumerable method

### PR DESCRIPTION
**Pull request by RoyNyaga**
No comments

### CODE REVIEW

**Comment by RoyNyaga** [comment]
I used the git flow concept and the library itself but did not make pull requests to merge to develop, once i finish with a feature branch, i merged to develop using the terminal command "git flow feature finish branch-name", it automatically merges the feature branch and deletes it.

File: enumerable.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    false
+  end
+
+  # Reverses the my All method
+  def my_none?
+    my_each { |value| return false if yield value }
+    true
+  end
+
+  def my_count
```
Comment (jofer86) [inline enumerable.rb]: This method works pretty well when a block is passed, however, we cannot pass an argument there si if I were to run the following code `[1,1,1].my_count(1)` it should return 3, but an error pops out. Please check that and make it work, once you are through with fixing that kindly resubmit.

**Review by jofer86 — CHANGES_REQUESTED** [review]
# :wave: :wave:  Hello!!! I'm Jorge and I'll be completing your code review today.
This was overall an excellent effort,  all of your code seems to be working like it should, with one exception though, please refer to the commented code to see what it is about.

Even though is a minor change I'm gonna have to ask you to resubmit for a code review, as the mistake I'm pointing is required to pass by the Microverse guidelines.

If you have any doubts contact me via Slack.

## Cheers, Jorge.

**Comment by RoyNyaga** [comment]
Fixed the bug in the my_count method as requested by the first reviewer. It can now take an argument without raising an error.

**Review by bolah2009 — APPROVED** [review]
## Awesome Job @RoyNyaga 

Good job implementing the changes suggested. :+1: 

Good luck with your next project. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Comment by RoyNyaga** [comment]
Thanks sir,

**Pull request merged**